docs/content/1.guide/0.getting-started.md

```diff
@@ -93,6 +97,71 @@ You can try it locally with:
 npm run preview
 ```
 
+::alert{type="info"}
+You can find more examples in the Nitro repository: [unjs/nitro/examples](https://github.com/unjs/nitro/tree/main/examples)
+::
+
+## From Scratch
```

Comment:
We shall advice to use starter template (`npx giget nitro`) b because for starting from scratch more than dependency is needed, particularly tsconfig. We might _explain_ it in next like what are basic files

Reply:
Ok, will remove it and add a section to explain files.